docs/running.rst

```diff
@@ -48,3 +48,63 @@ Your server can then be run by the simple command:
     honcho start
     
 On Windows, the ``--beat`` option may not be supported.
+
+Production installation of Wooey utilizing NGINX and uWSGI
```

Comment:
Could you change this to `Through NGINX and uWSGI` to be in sync with the rest of this page. This also doesn't cover production things like turning on SSL/using a separate database so it's not accurate to claim it.